packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -460,3 +480,36 @@ class PluginInterfaceResolution {
     return '<PluginInterfaceResolution ${plugin.name} for $platform>';
   }
 }
+
+/// A class representing pair of dartPluginClass and dartFileName used as metadata
```

Comment:
Why does this need to be an entire class, rather than just a Record?

Reply:
well, I guess it strictly because of my unawareness :) Never had a chance to use it, but now it is. Hope that's what you meant